dashboard/src/components/access/SSOLogout.vue

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2024 SkillTree
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+<script setup> 
```

Comment:
after another look there is no need for `SSOLogout.vue` component at all; logout can simply redirect to  the`/skills-login` route, then enhance `Login.vue` to optionally show the Login button, similar to what ` <Card v-if="oAuthProviders` card is doing; `SSOLogout.vue` and its route path and associated code can then be removed

Reply:
Removed SSOLogout.vue and modified code to redirect back to /skills-login route